docs/talkback-guidelines.md

```diff
@@ -0,0 +1,401 @@
+# TalkBack Guidelines
+
+### Table of Contents 
+
+- [Getting Started](#getting-started)
+- [Guidelines](#guidelines)
+   - [Basics](#basics)
+	- [Grouping Content](#grouping-content)
+	- [Live Regions](#live-regions)
+	- [Spoken Order](#spoken-order)
+	- [Focusing Views](#focusing-views)
+	- [Horizontal Scrolling](#horizontal-scrolling)
+	- [Appearing and Disappearing Elements](#appearing-disappearing)
+	- [Custom Views](#custom-views)
+- [Auditing](#auditing)
+  - [Questions to ask](#questions)
+  - [Manual Audit](#manual_auditing)
+  - [Automated Audit](#automated_auditing)
+  - [User Audit](#user_auditing)
+- [Further Reading](#further-reading)
+	
+## <a name="getting-started"></a>Getting Started
+“Making applications accessible not only ensures equal access to the roughly 1 billion people in the world with disabilities but also benefits people without disabilities by allowing them to customize their experiences.” - [Google Accessibility](https://www.google.com/accessibility/for-developers/)
+
+#### Governing Accessibility Principles 
+
+* Make the information on the screen as visible as possible. 
+* Always design the size of controls and touch areas inclusive of all use cases.
+* Provide as much support context and details for all views & actions. Images, buttons, lists and the actions they enable should have thorough, well-formatted descriptions that allow the accessibility APIs to deliver their intent seamlessly. 
+
+	For more details visit [Make apps more accessible
+](https://developer.android.com/guide/topics/ui/accessibility/apps.html).
+
+#### TalkBack
+This document will provide all the resources needed to improve an app's accessibility using TalkBack. If you have never worked with TalkBack you can visit the "Using TalkBack" document to get started.
+
+- [Using TalkBack](using-talback.md)
```

Comment:
The `Using TalkBack` link is not redirecting to the correct page for some reason. It keeps landing in 404 page.

Reply:
Oh! Thanks! Didn't spot that 💯